src/stdune/signal.ml

```diff
@@ -1,3 +1,7 @@
+external get_sigwinch : unit -> int option = "stdune_get_sigwinch"
```

Comment:
@rgrinberg you should be able to drop all the siginwch related changes, and instead unblock `SIGWINCH` in the UI thread